core/index.js

```diff
@@ -3,16 +3,40 @@
 console.log('hello World');
 
 //4. Read file(theory.md) here using fs module and console the results of operation.
+let fs = require('fs')
+let http =require('http')
+let server = http.createServer(handleRequest)
+function handleRequest(req,res){
+  fs.readFile(__dirname+"/theory.md",(err,data)=>{
+    if(err){
```

Comment:
If/Else is fine.
Alternatively
```js
if(err) return console.log(err);
res.end(data);
```
Above code looks much cleaner than adding If/Else clause.
If Error, it returns from there itself without moving to further code blocks, else automatically it evaluates res.end(data).